src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -604,44 +604,124 @@ internal BoundStatement BindLocalDeclarationStatement(LocalDeclarationStatementS
 
         private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSyntax node, DiagnosticBag diagnostics)
         {
-            var typeSyntax = node.Declaration.Type.SkipRef(out _);
+            var variableList = node.Declaration.Variables;
+            int variableCount = variableList.Count;
+
             bool isConst = node.IsConst;
+            bool IsUsing = node.UsingKeyword != default;
+            bool IsMultiple = variableCount > 1;
 
+            var typeSyntax = node.Declaration.Type.SkipRef(out _);
             bool isVar;
             AliasSymbol alias;
             TypeSymbol declType = BindVariableType(node.Declaration, diagnostics, typeSyntax, ref isConst, isVar: out isVar, alias: out alias);
 
-            // UNDONE: "possible expression" feature for IDE
+            Conversion iDisposableConversion = default;
+            MethodSymbol disposeMethod = default;
 
             LocalDeclarationKind kind = LocalDeclarationKind.RegularVariable;
             if (isConst)
             {
                 kind = LocalDeclarationKind.Constant;
             }
-            else if (node.UsingKeyword != default)
+            if (IsUsing)
```

Comment:
>if [](start = 12, length = 2)

Please keep as `else if`.

Reply:
I can't, the else on line 638 needs to catch all cases, including ones within the isConst on line 620.

Reply:
If the `else` at line 637 is removed, and the `else if` at line 623 is restored, the original structure is preserved.

---
In reply to: [206625160](https://github.com/dotnet/roslyn/pull/28920#discussion_r206625160) [](ancestors = 206625160)